server/routerlicious/packages/routerlicious/src/deli/index.ts

```diff
@@ -72,6 +72,12 @@ export async function deliCreate(
 	core.DefaultServiceConfiguration.deli.enableEphemeralContainerSummaryCleanup =
 		enableEphemeralContainerSummaryCleanup;
 
+	const ephemeralContainerSoftDeleteTimeInMs =
+		(config.get("deli:ephemeralContainerSoftDeleteTimeInMs") as number | undefined) ??
+		5 * 24 * 60 * 60 * 1000;
```

Comment:
nit: can you add a comment like `// 5 days`?

Also, I'm wondering if we should leave this as `-1` instead so that the change is made by explicit configuration. Not a strong opinion on this, so feel free to ignore.

Reply:
Changed to `-1` as default